docs/graphql/manual/schema/relationships/architecture.rst

```diff
@@ -0,0 +1,33 @@
+.. meta::
+   :description: Security, performance, schema integrity, and data federation with Hasura remote joins
+   :keywords: hasura, docs, data federation, remote relationship, remote join, remote schema
+
+.. _remote_joins_architecture:
+
+Architecture of remote joins
```

Comment:
We need to think about the title since it's not only for remote joins and it's also not really about architecture. @rikinsk and me were thinking about it and we're like 🤯 not able to come up with anything better yet :D

Reply:
I based this on @tirumaraiselvan 's talk on remote joins. I believe the concept of the join applies at the data federation level, e.g. you can also join with a field returned from an action.